### ISSUE TITLE
How to run Profiler?

### ISSUE THREAD

**Issue by tomk3003** [Camelcade/Perl5-IDEA#2311]
I just updated to 2020.2.3 and wanted to test profiling. But I cannot figure out how to run the tests or a script from my existing project with profiling enabled. The profiling option is always greyed out. Sorry If I missed something obvious.

PyCharm 2020.2.1 (Professional Edition)
Build #PY-202.6948.78, built on August 25, 2020
Runtime version: 11.0.8+10-b944.31 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-47-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 4029M
Cores: 8
Non-Bundled Plugins: net.seesharpsoft.intellij.plugins.csv, com.perl5, com.perl5.lang.embedded, com.perl5.lang.mojo, com.perl5.lang.tt2
Current Desktop: ubuntu:GNOME

**Comment by hurricup** [comment]
Profiling execution UI is a bit tricky:

![image](https://user-images.githubusercontent.com/2811330/93097834-c0f14080-f6ae-11ea-91ad-25efb821f239.png)

And there are configurations for profiling: 
![image](https://user-images.githubusercontent.com/2811330/93097947-e54d1d00-f6ae-11ea-9d53-87b86fbe3ab0.png)

**Comment by tomk3003** [comment]
A this is the Problem. I do not have an entry called "Profilers" under "Build. Execution, Deployment". Perhaps this is special to Pycharm?

**Comment by hurricup** [comment]
seems something wrong with Pycharm UI there. I see an action, but can't open it: 
![image](https://user-images.githubusercontent.com/2811330/93101316-c781b700-f6b2-11ea-9da7-6dc67a3460b3.png)

**Comment by tomk3003** [comment]
Yes it looks like the "Perl5 Profiler" settings do not have a parent node because the "Profilers" entry is missing.

**Comment by hurricup** [comment]
We can try to configure it manually:
1. Shutdown pycharm
2. Locate your application `config` directory, like this https://www.jetbrains.com/help/pycharm/tuning-the-ide.html#default-dirs
3. Create there a file with name `profilerRunConfigurations.xml` inside the `config` dir
4. Post content:
```
<application>
  <component name="ProfilerRunConfigurations">
    <profilerRunConfigurations>
      <item configurationTypeId="com.perl5.profiler.nytprof">
        <PerlProfilerConfigurationState name="NYTProf (immediately)">
          <option name="optimizerDisabled" value="false" />
          <option name="startupMode" value="BEGIN" />
        </PerlProfilerConfigurationState>
      </item>
      <item configurationTypeId="com.perl5.profiler.nytprof">
        <PerlProfilerConfigurationState name="NYTProf (init)">
          <option name="optimizerDisabled" value="false" />
          <option name="startupMode" value="INIT" />
        </PerlProfilerConfigurationState>
      </item>
      <item configurationTypeId="com.perl5.profiler.nytprof">
        <PerlProfilerConfigurationState name="NYTProf (end)">
          <option name="optimizerDisabled" value="false" />
          <option name="startupMode" value="END" />
        </PerlProfilerConfigurationState>
      </item>
      <item configurationTypeId="com.perl5.profiler.nytprof">
        <PerlProfilerConfigurationState name="NYTProf (manual)">
          <option name="optimizerDisabled" value="false" />
          <option name="startupMode" value="NO" />
        </PerlProfilerConfigurationState>
      </item>
    </profilerRunConfigurations>
    <knownConfigurationTypes>
      <id>com.perl5.profiler.nytprof</id>
    </knownConfigurationTypes>
  </component>
</application>
```
5. Start `PyCharm` and try again

**Comment by tomk3003** [comment]
I copied the file into the root of the config dir as well as into the "options" dir. 
Sadly this had no effect.

**Comment by tomk3003** [comment]
If I am interpreting the description of the features of the various JetBrains IDEs correctly, the profiler interface is only available for "IntelliJ IDEA Ultimate". Could you please verify this?

**Comment by hurricup** [comment]
Looks like this :( 
Seems it should be in:
- IDEA Ultimate
- CLion
- DataGrip :)
- Gogland
- RubyMine

And no: WebStorm, PhpStorm or Pycharm :(
Sorry, I didn't know that. 
I'll see what can be done there.

**Comment by hurricup** [comment]
Profiler should be available in all paid IDEs starting 2020.3. EAP cycle should start shortly. 
Sorry for misleading.

**Comment by tomk3003** [comment]
Thanks for the update, I will try again when 2020.3 is available
